src/components/navigation/navigation.component.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import { Outlet } from 'react-router-dom';
+import { Header } from '../header/header.component';
+import { MainContent } from '../main-content/main-content.component';
+import { Sidebar } from '../sidebar/sidebar.component';
+
+export function Navigation() {
```

Comment:
- [ ] This test is not working for me:

![image](https://user-images.githubusercontent.com/57421823/203671596-ddc40d76-f2af-4aa0-8df1-023d7c5c20e1.png)

Kindly check if this is working on your end.  If this is a bug, please fix it.  If it is not, kindly add the correct instructions about how to run it properly.